File: Database_Java/src/domain/UserList.java
package domain;

import java.util.ArrayList;

public class UserList {
    private ArrayList<User> userList;

    public UserList(){
        this.userList=new ArrayList<>();
    }

    public ArrayList<User> getUserList() {
        return userList;
    }



    public void addUser(User user){

        userList.add(user);
    }
    public void removeUser(User user){

          // if(userList.get(i).getUsername().equals(user.getUsername()))
            if (userList.contains(user)){
                userList.remove(user);
            }


    }

    @Override
    public String toString() {
        return "UserList{" +
                "userList=" + userList +
                '}';
    }
}

File: Bussiness Logic (Tier II)/CompareIT_API/Model/UserList.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CompareIT_API.Model
{
    public class UserList
    {
        public List<User> userList { get; set; }


        // shows the whole list with their properties. 
        /*  public void loadUsers() 
          {
              foreach (var user in userList)
              {
                  user.ToString();
              }
          }*/

        /* public void loadOneUser(String username)
         {
             foreach (var user in userList)
             {
                 if (username.Equals(user.getUsername()))
                 {
                     username.ToString();
                 }
             }
 
         }*/

        public void AddUser(User user)
        {
            userList.Add(user);
        }

        public void removeUser(User user)
        {
            if (userList.Contains(user))
            {
                userList.Remove(user);
            }
        }

        /*  public void loadWishes(String username)
          {
              foreach (var user in userList)
              {
                  if (username.Equals(user.getUsername()))
                  {
                      user.wish.wishList.ToString();
                  }
              }
          }*/



        /*    public User getUserTostring(User user)
            {
                user.ToString();
            }
        }*/
    }
}
